src/core/analytics.js

```diff
@@ -474,6 +472,7 @@ class Analytics {
                 }
                 // Transformation server access denied
                 case 404: {
+                  logger.warn(`System failure`);
```

Comment:
The warning message could be more explicit.
```suggestion
                  logger.warn(`[DMT]:: Transformation server access is denied. The configuration data seems to be out of sync. Sending untransformed event to the destination.`);
```

Reply:
chore: error message update